backend/amd64/emit.mlp

```diff
@@ -1133,7 +1133,7 @@ let emit_instr fallthrough i =
       | S_macosx | S_win64 -> () (* with LLVM/OS X and MASM, use the text segment *)
       | _ -> D.section [".rodata"] None []
       end;
-      D.align 4;
+      D.align ~data:false 4;
```

Comment:
This should be `~data:true` I think, we're in a `.rodata` section here.

Reply:
Ah, I had seen the emission of instructions above and assumed it was code but yeah its setting the section to data.